src/hooks/useMarketsData.ts

```diff
@@ -52,22 +52,42 @@ export const useMarketsData = (
 } => {
   const allPerpetualMarkets = useSelector(getPerpetualMarkets, shallowEqual) || {};
   const allAssets = useSelector(getAssets, shallowEqual) || {};
-  const sparklineData = usePerpetualMarketSparklines();
+  const sevenDaysSparklineData = usePerpetualMarketSparklines();
```

Comment:
@jaredvu correct me if i'm wrong but I think this is the part that would slow down the application. pulling historic data for 7 days worth is going to impact the loading time of the app due to the endpoint this uses, unfortunately. 

@DavideSegullo would we be able to revert the sparklines code completely? right now i'm still seeing historic data in the testnet deployment.
<img width="440" alt="Screenshot 2024-04-29 at 10 37 14 AM" src="https://github.com/dydxprotocol/v4-web/assets/37092291/754ff028-d7c3-46c9-b034-1c4a5a29e690">

Reply:
However, this service was already there, and should already be used in prod, right? Do you mean can I remove the code on the sparkline at 7 days?

Reply:
I just made it generic with some params